bitnami/mariadb-galera/README.md

```diff
@@ -157,6 +157,12 @@ The following table lists the configurable parameters of the MariaDB Galera char
 | `metrics.serviceMonitor.namespace`   | Optional namespace which Prometheus is running in                                                                                                           | `nil`                                                             |
 | `metrics.serviceMonitor.interval`    | How frequently to scrape metrics (use by default, falling back to Prometheus' default)                                                                      | `nil`                                                             |
 | `metrics.serviceMonitor.selector`    | Default to kube-prometheus install (CoreOS recommended), but should be set according to Prometheus install                                                  | `{prometheus: "kube-prometheus"}`                                 |
+| `ssl.enabled`                        | Setup and use SSL for MySQL connections                                                                                                                     | `false`                                                           |
+| `ssl.secret`                         | Name of the secret containing the SSL certificates                                                                                                          | mysql-ssl-certs                                                   |
+| `ssl.certificates[0].name`           | Name of the secret containing the SSL certificates                                                                                                          | `nil`                                                             |
```

Comment:
You may need to update the field description

Reply:
Hi, I'm going to change MySQL to MariaDB.

Reply:
the `ssl.secret` and `ssl.certificates[0].name` have the exact same description.